app/controllers/v1/actions_controller.rb

```diff
@@ -0,0 +1,19 @@
+class V1::ActionsController < V1::BaseController
+
+  def create
+    activity = Activity.find_by(template_id: params[:template_id])
+    action = Action.create(person: person_from_params, activity: activity)
+    if action.valid?
+      output = { success: true }
+    else
+      output = { error: 'could not create action' }
+    end
+    render json: output
+  end
+
+  private
+
+  def person_from_params
+    Person.find_by(email: params[:email]) || Person.find_by(phone: params[:phone])
```

Comment:
Won't this still do a search even if email is an empty string?  Would something like this be better?

`Person.where('email = :email or phone = :phone', params.slice(:email, :phone) ).first`

I'll merge anyway, but would be a good optimization.